src/schemas/xspec.rnc

```diff
@@ -136,6 +149,7 @@ function-call =
 	## parameters passed to it. 
 	element call {
 		common-attributes,
+		##The qualified name of the function which should get called.
```

Comment:
As described in https://github.com/xspec/xspec/pull/115#issuecomment-292457562, please add a space after ```##``` to make it consistent with the rest of the file, i.e.: 
```
## The qualified name of the function which should get called.
```